js/id/renderer/features.js

```diff
@@ -360,7 +384,10 @@ iD.Features = function(context) {
 
     features.isHiddenChild = function(entity, resolver, geometry) {
         if (!_hidden.length) return false;
-        if (!entity.version || geometry === 'point') return false;
+        if ((!context.indoor().enabled() && entity.isNew()) || geometry === 'point') return false;
+
+        //hide vertices with different level
+        if (geometry === 'vertex' && context.indoor().enabled() && _features.indoor_different_level.filter(entity, resolver, geometry)) return true;
```

Comment:
Default behaviour is iD shows new entities regardless the feature filter. But here i need to hide different level always.. 

Also for the code clarity i replaced `!entity.version` for `entity.isNew()` – is it correct?